docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -20,50 +20,48 @@ date: "2022-10-11"
 cost: "0"
 ---
 
-This tutorial will show you how to build a mock robot using just your personal laptop so you can:
+In this tutorial you will build a mock robot to learn how to configure robots with Viam.
+You do not need any hardware to do this tutorial.
 
-- Learn how to [configure](../../../manage/configuration/) robots with Viam.
-- Try using [Viam](/viam/) without any robotic hardware.
+Follow this tutorial to set up and control a robot with a `fake` [arm](/components/arm/fake/), [board](/components/board/), and [motor](/components/motor/), and an additional mock {{< glossary_tooltip term_id="part" text="sub-part" >}} with a [motor](/components/motor/).
+These `fake` components interact with Viam like real hardware but do not physically exist.
 
-Most Viam [components](../../../components/) have a _fake_ model that you can use for testing.
-These fake components interact with Viam like real hardware but do not actually exist.
-
-In this tutorial, you will set up, control, and program a mock robotic arm and a mock sub-part with a motor using fake components.
-
-## Requirements
+## Set up a mock robot
 
 You'll need the following hardware and software for this tutorial:
 
-- A laptop or desktop running Linux or macOS
+- A computer running Linux or macOS
 - [Go](https://go.dev/doc/install) or [Python 3.9+](https://www.python.org/downloads/)
-- A code editor of your choice
 
-## Set up a mock robot
+If you don't already have an account, create a free one.
+Sign up on [the Viam app](https://app.viam.com).
```

Comment:
I didn't really mean use the words quote on quote. In the first sentence you have no context for what account you're talking about.

```suggestion
If you don't already have a Viam account, sign up for one on [the Viam app](https://app.viam.com).
```